client/Lykke.Service.BlockchainSettings.Client/HttpClientGenerator/IBlockchainSettingsControllerFactory.cs

```diff
@@ -1,9 +1,14 @@
 ﻿using System.Net.Http;
+using Lykke.HttpClientGenerator.Caching;
 
 namespace Lykke.Service.BlockchainSettings.Client.HttpClientGenerator
 {
     public interface IBlockchainSettingsControllerFactory
```

Comment:
Looks like, this should be named BlockchainSettingsClientFactory

Reply:
fixed